tree-construction/doctype01.dat

```diff
@@ -368,14 +368,17 @@
 
 #data
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Frameset//EN"
-"http://www.w3.org/TR/xhtml1/DTD/xhtml1-frameset.dtd">
```

Comment:
There was no test for the `<p><table>` behavior in limited quirks mode.

Reply:
Can you add this as a new test, not replacing the existing one?

Reply:
Done!